test/distributed_tests_utils.jl

```diff
@@ -104,17 +104,21 @@ function run_distributed_latitude_longitude_grid(arch, filename)
 
     distributed_grid = ImmersedBoundaryGrid(distributed_grid, GridFittedBottom(bottom_height))
     model = run_distributed_simulation(distributed_grid)
-
+    
     η = reconstruct_global_field(model.free_surface.η)
     u = reconstruct_global_field(model.velocities.u)
     v = reconstruct_global_field(model.velocities.v)
     c = reconstruct_global_field(model.tracers.c)
 
+    # Check also that the bottom height is reconstructed correctly!
+    b = reconstruct_global_field(model.grid.immersed_boundary.bottom_height)
+
     if arch.local_rank == 0
         jldsave(filename; u = Array(interior(u, :, :, 10)),
                           v = Array(interior(v, :, :, 10)), 
                           c = Array(interior(c, :, :, 10)),
-                          η = Array(interior(η, :, :, 1))) 
+                          η = Array(interior(η, :, :, 1)),
+                          b = Array(parent(b))[:, :, 1]) 
```

Comment:
is there a reason not to save `parent` for all`?

Reply:
hmm, I have removed that `b` field.

Reply:
we could save parent.